src/documentation/programming-guide.md

```diff
@@ -684,8 +684,8 @@ PCollection<Integer> wordLengths = words.apply(
   "ComputeWordLengths",                     // the transform name
   ParDo.of(new DoFn<String, Integer>() {    // a DoFn as an anonymous inner class instance
       @ProcessElement
-      public void processElement(ProcessContext c) {
-        c.output(c.element().length());
+      public void processElement(@Element String word, OutputReceiver out) {
```

Comment:
`OutputReceiver<String>`

Reply:
Is this backwards compatible to previous releases of Beam (2.x)  if not, maybe we should say explictly so.

Reply:
@iemejia yes it is. ProcessContext still exists as before, and cannot be removed until Beam 3.0 for compatibility reasons.

Reply:
@reuvenlax I am not talking about `ProcessContext` I refer to `@Element` and the new annotations not included in previous versions of 2.x. Users should be aware of this given that there is not backwards compatibility on those.